Homework-1-NodeJS-Fundamentals/task1/index.js

```diff
@@ -0,0 +1,20 @@
+import { Transform } from 'stream';
+
+const reverseString = str => str.split("").reverse().join("");
+
+const removeLineBreaks = str => str.replace(/(\r\n|\n|\r)/gm, "");
```

Comment:
I would use `trim()` but thanks for thinking about different types of line separators.